index.js

```diff
@@ -1,7 +1,10 @@
 'use strict';
 /* eslint-disable import/no-dynamic-require */
-module.exports = () => {
-	const plugins = require(`./plugins/best-match`)
+const modulePlugins = require('./plugins/module');
```

Comment:
Perhaps rename the file to `modules`, and import it into a `determineModulesPlugin()` variable?